aspnetcore/performance/performance-best-practices/samples/3.0/Controllers/MyFirstController.cs

```diff
@@ -27,11 +27,25 @@ public class GoodStreamReaderController : Controller
         [HttpGet("/contoso")]
         public async Task<ActionResult<ContosoData>> Get()
         {
-            return await JsonSerializer.DeserializeAsync<ContosoData>(Request.Body);
+            var json = await new StreamReader(Request.Body).ReadToEndAsync();
+            
+            return await System.Text.Json.JsonSerializer.Deserialize<ContosoData>(json);
```

Comment:
Remove the full namespace.

Reply:
Error	CS1061	'ContosoData' does not contain a definition for 'GetAwaiter' and no accessible extension method 'GetAwaiter' accepting a first argument of type 'ContosoData' could be found

Reply:
Just updated. Thanks!